docs/gl_objects/merge_requests.rst

```diff
@@ -78,8 +78,8 @@ List MRs for a project::
 
 You can filter and sort the returned list with the following parameters:
 
-* ``state``: state of the MR. It can be one of ``all``, ``merged``, ``opened``
-  or ``closed``
+* ``state``: state of the MR. It can be one of ``all``, ``merged``, ``opened``,
+   ``closed`` or ``locked``
 * ``order_by``: sort by ``created_at`` or ``updated_at``
 * ``sort``: sort order (``asc`` or ``desc``)
```

Comment:
We could also link to the upstream docs, because the list of filter/sort attributes grows with GitLab versions. See https://docs.gitlab.com/ee/api/merge_requests.html#list-merge-requests